resources/zib/zib-AdvanceDirective.xml

```diff
@@ -0,0 +1,218 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="zib-AdvanceDirective" />
+  <url value="http://nictiz.nl/fhir/StructureDefinition/zib-AdvanceDirective" />
+  <name value="ZibAdvanceDirective" />
+  <title value="zib AdvanceDirective" />
+  <status value="draft" />
+  <publisher value="Nictiz" />
+  <contact>
+    <name value="Nictiz" />
+    <telecom>
+      <system value="email" />
+      <value value="info@nictiz.nl" />
+      <use value="work" />
+    </telecom>
+  </contact>
+  <description value="A living will is a verbal or written description of the patient’s wishes with regard to future medical action or end of their life. A living will is mainly used for situations in which the patient is no longer able to speak about these decisions with their healthcare provider." />
+  <purpose value="This Consent resource represents the Dutch [zib ('Zorginformatiebouwsteen', i.e. Health and Care Information Model) AdvanceDirective-v3.1.1(2020EN)](https://zibs.nl/wiki/AdvanceDirective-v3.1.1(2020EN))." />
+  <copyright value="Copyright and related rights waived via CC0, https://creativecommons.org/publicdomain/zero/1.0/. This does not apply to information from third parties, for example a medical terminology system. The implementer alone is responsible for identifying and obtaining any necessary licenses or authorizations to utilize third party IP in connection with the specification or otherwise." />
+  <fhirVersion value="4.0.1" />
+  <mapping>
+    <identity value="zib-advancedirective-v3.1.1-2020EN" />
+    <uri value="https://zibs.nl/wiki/AdvanceDirective-v3.1.1(2020EN)" />
+    <name value="zib AdvanceDirective-v3.1.1(2020EN)" />
+  </mapping>
+  <kind value="resource" />
+  <abstract value="true" />
+  <type value="Consent" />
+  <baseDefinition value="http://hl7.org/fhir/StructureDefinition/Consent" />
+  <derivation value="constraint" />
+  <differential>
+    <element id="Consent">
+      <path value="Consent" />
+      <short value="AdvanceDirective" />
+      <definition value="Root concept of the LivingWill information model. This concept contains all data elements of the LivingWill information model." />

```

Comment:
Based on https://informatiestandaarden.nictiz.nl/wiki/FHIR:V1.0_FHIR_Profiling_Guidelines_R4#StructureDefinitions (11.2.1.2, second bullet), this definition should not be taken from the zib, as it does not provide meaningful additional information.